src/actions/createRoom.js

```diff
@@ -103,6 +103,12 @@ export const onCreateRoom = payload => (dispatch, getState) => {
       dispatch(goToChannel(channelId))
     })
     .catch(err => {
+      if (err.message === 'Name must not be empty') {
+        dispatch({
+          type: types.HANDLE_EMPTY_GROUP_NAME_ERROR,
+          payload: err.message,
+        })
+      }
```

Comment:
we should display all errors from the server in the frontend not pick single errors resposnes and handle them manually

Reply:
this will be fixed in the backend. also, the error string is translated in the backend, so this will only work for english users :D